iceoryx_platform/win/include/iceoryx_platform/mqueue.hpp

```diff
@@ -57,7 +57,7 @@ inline ssize_t mq_receive(mqd_t mqdes, char* msg_ptr, size_t msg_len, unsigned i
     return 0;
 }
 
-inline ssize_t
+inline mqd_t
 mq_timedreceive(mqd_t mqdes, char* msg_ptr, size_t msg_len, unsigned int* msg_prio, const struct timespec* abs_timeout)
```

Comment:
Please also revert this line. It's the last one.

Reply:
> Please also revert this line. It's the last one.

@elBoberido Sorry, I didn't see it.

Reply:
@elBoberido canceled the changes